packages/@ourworldindata/grapher/src/mapCharts/MapChart.tsx

```diff
@@ -725,6 +785,45 @@ class ChoroplethMap extends React.Component<{
         )
     }
 
+    @computed get globeWidth(): number {
+        return (this.bounds.width - 300) / 2
+    }
+
+    @computed get globeHeight(): number {
+        return (this.bounds.height - 150) / 2
+    }
+
+    @computed get globeScale(): number {
+        if (this.bounds.height < this.bounds.width) {
```

Comment:
Noting on line [773](https://github.com/owid/owid-grapher/pull/3057/files#diff-1e9d0a7908d5769c4657f34c3286d9f969805a0b9e432f09eb889103eb5b9756R773) that `this` is destructured and that pattern is in the file elsewhere, could be followed here